src/components/DataGrid/DataGrid.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { DataGrid as MuiDataGrid, DataGridProps } from '@mui/x-data-grid';
+
+import theme from '../../theme';
+import { makeStyles } from '../../theme';
+import Box from '../Box';
+import styled from '../../theme/styled';
+
+export interface Props extends DataGridProps {
```

Comment:
should do `export interface DataGridProps extends MuiDataGridProps` (import native props with the Mui prefix)